### PR TITLE
Add: Expand place options

### PR DESCRIPTION
**Pull request by 1kuemmel1**
**When merged this pull request will:**
- Add a new placing option: tilt
- Changed the handling of rotating and tilting of an object
- Use ACE3 Mousehint for release action
- Remove the old addAction entry
- Adaptation of stringtable.xml
- Changed handling of object status in database (save/load)
   - use ```getPosWorld ``` and ```setPosWorld``` instead of ```getPosASL``` and ```setPosASL```
   - added some comments
   - handle ```setVectorDirAndUp``` on database save and load

**Final test:**
- [x] local
- [x] server
- [x] database handling of objects (save/load)

**Screenshots**
placeing_new_full
placing_new_releaseplacing_new_hint

### CODE REVIEW

**Comment by Vdauphin** [comment]
Hello, very good idea but you also need to make it persistent ([/core/fnc/db/save.sqf#L133](https://github.com/Vdauphin/HeartsAndMinds/blob/master_daily/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/db/save.sqf#L133) and [/core/fnc/db/load.sqf#L287](https://github.com/Vdauphin/HeartsAndMinds/blob/master_daily/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/db/load.sqf#L287)).

I notice you use a lot of brackets. I am following this coding guidelines for those : https://ace3mod.com/wiki/development/coding-guidelines.html#55-brackets-around-code. Can you also follow it ?
For  exemple : `[(localize "STR_BTC_HAM_LOG_PLACE_RELEASE"),""] call ace_interaction_fnc_showMouseHint; //Release`
Juste write : `[localize "STR_BTC_HAM_LOG_PLACE_RELEASE", ""] call ace_interaction_fnc_showMouseHint; //Release`

Thanks see you soon !

**Comment by 1kuemmel1** [comment]
Hi, I've updated some thinks.

> I notice you use a lot of brackets. I am following this coding guidelines for those : https://ace3mod.com/wiki/development/coding-guidelines.html#55-brackets-around-code. Can you also follow it ?

Sorry for this. 
Did I make it correct now?

I've modified the *loadObjectStatus.sqf* and *saveObjectStatus.sqf* to handle vector settings.

Database saved:
beforedb_save

Database loaded:
afterdb_save

Update first comment in PR.

**Comment by Vdauphin** [comment]
This looks perfect ! I will review it soon. Just some questions :
- Why do you use ```getPosWorld ``` and ```setPosWorld``` instead of ```getPosASL``` and ```setPosASL``` ?
- Does the `setVectorDirAndUp` is correctly broadcasted between client on dedicate server? I noticed for C4 on suicider there was not even if the Wiki say yes :/ (https://github.com/Vdauphin/HeartsAndMinds/pull/282).
- For comment, you don't need to explain each line from my point of view but do it as you like. For coding guidelines you can refer you to this : https://ace3mod.com/wiki/development/coding-guidelines.html#54-comments-in-code

**Comment by 1kuemmel1** [comment]
> Why do you use getPosWorld and setPosWorld instead of getPosASL and setPosASL ?

Because *PositionWorld* referenced to model center. This is better (more accurate) for ```setVectorDirAndUp```. And BIS said: *But new to Arma 3 command getPosWorld is the fastest of them all.* 😄  (see: [Code Optimisation](https://community.bistudio.com/wiki/Code_Optimisation) see: *Position World is the fastest*)

> Does the setVectorDirAndUp is correctly broadcasted between client on dedicate server? I noticed for C4 on suicider there was not even if the Wiki say yes :/ (#282).

On a locally hosted dedicated server, everything seems to be fine. I'm going to do a little test on a dedicated server soon. (Tomorrow or so).

**Comment by 1kuemmel1** [comment]
**Update:**
After a little test on a dedicated server, I can say: it's going quite well so far.

*Saved to database:*
savedb_dedi

*After server restart (loaded from database):*
loaddb_dedi

**Comment by 1kuemmel1** [comment]
Ahh sorry, I hit the wrong button.😕 😃

**Comment by Vdauphin** [comment]
Good job ! looks like it works prefectly!

I forgot to ask you to increment the mission version to prevent data base conflict with previous version (`1.17`). To do so just write `1.171` in https://github.com/Vdauphin/HeartsAndMinds/blob/master_daily/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L4

**Comment by 1kuemmel1** [comment]
> I forgot to ask you to increment the mission version to prevent data base conflict with previous version (1.17).

done

File: =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/log/place.sqf [hunk 1]
```diff
@@ -1,44 +1,69 @@
...
+
+[{
+	params ["_arguments", "_idPFH"];
+	if (!alive player || player getVariable ["ACE_isUnconscious",false] || !btc_log_placing) then {
+
+		btc_log_placing_obj enableSimulation true;
+		detach btc_log_placing_obj;
+
+		//save to DB
+		[btc_log_placing_obj] call btc_fnc_db_saveObjectStatus;
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/log/place.sqf]: @1kuemmel1  Why do you use `btc_fnc_db_saveObjectStatus` here ? You are doing nothing, right ?

Comment (1kuemmel1) [reply =BTC=co@30_Hearts_and_Minds.Altis/core/fnc/log/place.sqf]: Why?... actually no idea. But you be right it's useless there. Just remove it.

**Pull request merged**